src/server.go

```diff
@@ -101,25 +101,24 @@ func (a *App) ServeHTTP(w http.ResponseWriter, r *http.Request) {
   switch r.Method {
   case "GET", "HEAD":
     rec := a.GetRecord(key)
-    var volume string
+    var remote string
     if rec.deleted == SOFT || rec.deleted == HARD {
       if a.fallback == "" {
         w.Header().Set("Content-Length", "0")
         w.WriteHeader(404)
         return
-      } else {
-        // fall through to fallback
-        volume = a.fallback
       }
+      // fall through to fallback
+      remote = fmt.Sprintf("http://%s%s", a.fallback, key)
```

Comment:
I don't like this. What if key has non URL chars in it?

Reply:
I think whatever was sent for the URL in the original GET request should be fine to pass through to the fallback endpoint.  I think base64 encoding the name only makes sense when writing a file out to disk (don't want slashes in file name).  Also, `key2path` includes the folder structure, which the fallback shouldn't have to deal with.